validator/rules_src/binary.py

```diff
@@ -0,0 +1,42 @@
+from validator.rules_src import Rule
+
+
+class Binary(Rule):
+    """
+    The field under validation must be an String
```

Comment:
Please change this line to:
`The field under validation must be a binary number`